src/System Application/App/Azure AD User Management/src/User sync/AzureADUserSyncImpl.Codeunit.al

```diff
@@ -495,7 +496,7 @@ codeunit 9029 "Azure AD User Sync Impl."
             if UpdatedSuccessfully then
                 NumberOfSuccessfulUpdates += 1
             else
-                Session.LogMessage('0000BPA', GetLastErrorCallStack, Verbosity::Normal, DataClassification::SystemMetadata, TelemetryScope::ExtensionPublisher, 'Category', UserSetupCategoryTxt); // to be taken out;
+                Session.LogMessage('0000BPA', StrSubstNo(FailedToUserUpdateTxt, AzureADUserUpdate."User Security ID", AzureADUserUpdate."Authentication Object ID", GetLastErrorCallStack), Verbosity::Error, DataClassification::SystemMetadata, TelemetryScope::All, 'Category', UserSetupCategoryTxt);
```

Comment:
```suggestion
                Session.LogMessage('0000BPA', StrSubstNo(FailedToUserUpdateTxt, GetLastErrorCallStack), Verbosity::Error, DataClassification::SystemMetadata, TelemetryScope::All, 'Category', UserSetupCategoryTxt);
```

We should not emit User ID into Telemetry, it's EUPI. Only the Error callstack is helpful enough